Guidelet/GuideletLoadable.py

```diff
@@ -265,13 +281,14 @@ class GuideletTest(ScriptedLoadableModuleTest):
   """
   #TODO add common test utility methods here
 
+  # -----------------------------------------------------
   def setUp(self):
-    """ Do whatever is needed to reset the state - typically a scene clear will be enough.
-    """
+    # Do whatever is needed to reset the state - typically a scene clear will be enough.
```

Comment:
Python standard method documentation format (https://www.python.org/dev/peps/pep-0257/).
Try for example this in the Slicer console:
help(slicer.util.getNode)